lib/libyui_client/app.rb

```diff
@@ -171,5 +172,29 @@ def tree(filter)
     def wizard(filter)
       Widgets::Wizard.new(@widget_controller, FilterExtractor.new(filter))
     end
+
+    # Returns client side libyui REST API version
+    # @return libyui client REST API version
+    def client_api_version
+      API_VERSION
+    end
+
+    # Returns server side libyui REST API version
+    # @return libyui client REST API version
```

Comment:
Just a minor thing. Could you please change 'client' to 'server' here.

Reply:
oops, copy-paste issue. Thanks!